### PR TITLE
Itsasu 269 migrate have u claimed covid grant

### PR DESCRIPTION
**Pull request by llmikeyj**

### CODE REVIEW

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html [hunk 1]
```diff
@@ -62,23 +68,37 @@
...
+
+@mainTemplate(title = messages("covid_19_claim_check.title"), optForm = Some(covid19ClaimCheckForm)) {
+
+    <div class="govuk-form-group @{if (covid19ClaimCheckForm("yes-no").error.isDefined) "govuk-form-group--error"}" @ariaDescribedBy>
+
+        <legend>
+            <h1 class="govuk-heading-l">
+                @messages("covid_19_claim_check.title")
+            </h1>
+        </legend>
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:85]: Unneccesry addition, our new radio helper is to make these pages simpler:
Structure of this page should be like:
`
mainTemplate
  h1
  content
  form
    radiohelper
    continuebutton
`

Comment (AlexRimmerHMRC) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:85]: We aren't needing to add the extra stuff to make the red border appear full page, we have discussed this with Jess and the frontend community and they agree with this approach, the red border should only cover from the legend to the end of the radio buttons

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html [hunk 2]
```diff
@@ -62,23 +68,37 @@
...
-            inline = true
-        )
-
-        <button id="continue-button" class="button" type="submit">@messages("base.continue")</button>
+            isPageHeading = false,
+            inline = true,
+            radioItems = Seq(
+                RadioItem(value = Some(option_yes), content = HtmlContent(messages("base.yes"))),
+                RadioItem(value = Some(option_no), content = HtmlContent(messages("base.no")))),
+            legend = "not used but also mandatory?")
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:98]: Legend is used, this is read out by screen readers when they go into the radio buttons, should match the h1 of the page

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html [hunk 3]
```diff
@@ -62,23 +68,37 @@
...
-            ),
-            parentForm = covid19ClaimCheckForm,
-            inline = true
-        )
-
-        <button id="continue-button" class="button" type="submit">@messages("base.continue")</button>
+            isPageHeading = false,
+            inline = true,
+            radioItems = Seq(
+                RadioItem(value = Some(option_yes), content = HtmlContent(messages("base.yes"))),
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:96]:
```suggestion
                RadioItem(value = Some(option_yes), content = Text(messages("base.yes"))),
```
HtmlContent not necessary for the purposes of this radio button

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html [hunk 4]
```diff
@@ -62,23 +68,37 @@
...
+                RadioItem(value = Some(option_no), content = HtmlContent(messages("base.no")))),
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:97]:
```suggestion
                RadioItem(value = Some(option_no), content = Text(messages("base.no")))),
```
HtmlContent not necessary for the purposes of this radio button

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html (hunk 1 above)
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:85]: Okay

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html (hunk 2 above)
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:98]: Okay

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html (hunk 3 above)
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:96]: Okay

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html (hunk 4 above)
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html:97]: Okay

**Pull request merged**